pkg/util/containers/v2/metrics/docker/collector_docker.go

```diff
@@ -0,0 +1,140 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
nit: we can simplify the file names since we moved each collector in a sub-package, `mv collector_docker.go collector.go` (it applies to every collector)

Reply:
Done!